spec/lib/nationbuilder/client_spec.rb

```diff
@@ -50,6 +50,7 @@
     it 'should handle a parametered GET' do
       VCR.use_cassette('parametered_get') do
         response = client.call(:basic_pages, :index, site_slug: 'organizeralexandreschmitt', limit: 11)
+        expect(response['status_code']).to eq(200)
```

Comment:
While we're at it, I think we should add test coverage of this to the other HTTP methods used here as well.